components/LoginForm/LoginForm.js

```diff
@@ -54,28 +55,36 @@ class LoginScreen extends React.Component {
     );
   }
 
-selectImg() {
-  ImagePicker.requestCameraRollPermissionsAsync()
-    .then(response => {
-      if (response.granted === true) {
-        ImagePicker.launchImageLibraryAsync()
-          .then(response => this.setState({ image: response.uri }))
+  selectImg() {
+    ImagePicker.requestCameraRollPermissionsAsync()
+      .then(response => {
+        if (response.granted === true) {
+          ImagePicker.launchImageLibraryAsync()
+            .then(response => this.uploadImg(response.uri, "Test-Image")
```

Comment:
This is where the image is sent back! Nice! Is this simple to implement on other components as well?

Reply:
Yes, its not too complex at all! We should be able to invoke this method anywhere we need to upload images. I still need to change the 'Test-Image' name to be unique with each image. Right now it is just overwriting the previous one because of the same name.